src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -1,5 +1,6 @@
 (ns quo2.components.notifications.activity-logs
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as str]
```

Comment:
We are using `string` as alias in most of the files, better to stick to the same alias for all the files using this namespace to keep consistency